pilot/test/integration/access.go

```diff
@@ -87,11 +87,11 @@ func (a *accessLogs) check(infra *infra) error {
 				logs := util.FetchLogs(client, pod, ns, container)
 
 				if strings.Contains(logs, "segmentation fault") {
-					return fmt.Errorf("segmentation fault %s", pod)
+					return fmt.Errorf("segmentation fault %s log: %s", pod, logs)
```

Comment:
Good enough for now - but in a separate PR maybe just dump the log to a file - regardless of result, just like the e2e tests are doing. We want to improve consistency - eventually all this code should be moved to common (or maybe we can reuse the same code as the e2e framework ).

But one step at a time, let's get this in first.